### ISSUE TITLE
Do you have data for the training set and the test set?

### ISSUE THREAD

**Issue by 1315975895** [ziyangyeh/Auto-Orth#10]
There is no directly usable dataset in the project to get the project running
